### PR TITLE
TUNIC: Add ice grapple logic to get to gauntlet

### PR DESCRIPTION
**Pull request by ScipioWright**
## What is this fixing or adding?
For NMG logic, now includes ice grappling into the elevator hole to get to gauntlet. I don't know who is so eager to get to gauntlet that they would do this, but this is for you, weird guy.

## How was this tested?
https://discord.com/channels/731205301247803413/1224388891948093540/1247356605465362434
And also running gen and it genning, and running tests.

## If this makes graphical changes, please attach screenshots.
N/A

### CODE REVIEW

**Review by nicholassaylor — APPROVED** [review]
Changed LGTM, godspeed weird guy!

File: worlds/tunic/rules.py [hunk 1]
```diff
@@ -312,7 +312,9 @@ def set_location_rules(world: "TunicWorld", ability_unlocks: Dict[str, int]) ->
 
     # Swamp
     set_rule(multiworld.get_location("Cathedral Gauntlet - Gauntlet Reward", player),
-             lambda state: state.has(laurels, player) and state.has(fire_wand, player) and has_sword(state, player))
+             lambda state: (state.has(fire_wand, player) and has_sword(state, player))
```
Comment (Exempt-Medic) [inline worlds/tunic/rules.py:315]: I don't believe these parens are necessary
```suggestion
             lambda state: state.has(fire_wand, player) and has_sword(state, player)
```

Comment (ScipioWright) [reply worlds/tunic/rules.py:315]: They're not, I just feel weird not putting them in -- makes the intent much clearer imo

**Review by Exempt-Medic — APPROVED** [review]
LGTM, generates just fine still.

**Pull request merged**